ethereum/statetransition/src/main/java/tech/pegasys/teku/statetransition/util/AbstractIgnoringFutureHistoricalSlot.java

```diff
@@ -46,12 +48,13 @@ public AbstractIgnoringFutureHistoricalSlot(
     this.spec = spec;
     this.futureSlotTolerance = futureSlotTolerance;
     this.historicalSlotTolerance = historicalSlotTolerance;
+    this.slotsInEpoch = UInt64.valueOf(spec.getGenesisSpec().getSlotsPerEpoch());
```

Comment:
not a fan of this.

Reply:
but ok :)

Reply:
Yeah i've noted this thing in PR description
I don't want to call spec.onSlot(slot).getSlotsPerEpoch on every slot when it's the same result all life